code/service/src/test/scala/com/namely/chiefofstate/readside/Cancellable.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 Namely Inc.
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+package com.namely.chiefofstate.readside
+
+import java.util.concurrent.FutureTask
+import scala.concurrent.{ ExecutionContext, Future, Promise }
+import scala.util.Try
+
+class Cancellable[T](executionContext: ExecutionContext, todo: => T) {
```

Comment:
where is this class used?

Reply:
I'm using it in one of my tests to properly clean up my future